### DISCUSSION TITLE
FluidSynth now integrated in JJazzLab 4

### DISCUSSION THREAD

**Discussion by jjazzboss** [FluidSynth/fluidsynth#1293] [Show and tell]
Just a message to say thank you (and merry christmas!) to the FluidSynth devs. The brand new JJazzLab 4 release now integrates the Fluidsynth lib, using the new Java 17 Panama preview project. Feedback from users is rather good for now.

**Comment by derselbst** [comment]
Thanks Jerome, glad to hear! I've just added JJazzLab to our [Applications](https://github.com/FluidSynth/fluidsynth/wiki/Applications) page.

**Comment by jjazzboss** [comment]
thank you. If you need a screen capture for your showcase, here it is.
![JJazzLab4-fullWithNoteEditor](https://github.com/FluidSynth/fluidsynth/assets/54775378/11e5ed8f-5c6c-4c51-a575-9c94d0468f43)

**Comment by jjazzboss** [comment]
I have a question regarding Linux packages (which I'm not familiar with...).

I'd like to give clear instructions to my Linux users, eg you need FluidSynth >= 2.1, but for now I'm confused between the FluidSynth package version and libfluidsynth version, which can be different. How does this work ?

For example, I checked a few distros and thought that libfluidsynth3 was the standard. But I got errors from Debian 11 users who actually use libfluidsynth.2.so, as part of the FluidSynth 2.1.7 package. When JJazzLab checked that version was >= 2.1 via API call fluid_version(), it failed, though user has installed FluidSynth 2.1.7...

**Reply by derselbst** [reply]
In `libfluidsynth3` the 3 indicates the SOVERSION used for ABI / API compatibility. We try to keep the number of ABI breaks low, but last time it was necessary. Hence I recommend you to require fluidsynth >= 2.2.0.

**Reply by jjazzboss** [reply]
thanks!
